pkg/pquerier/collector.go

```diff
@@ -72,16 +74,25 @@ func mainCollector(ctx *selectQueryContext, responseChannel chan *qryResults) {
 				ctx.errorChannel <- err
 				return
 			}
-
-			if res.IsServerAggregates() {
-				aggregateServerAggregates(ctx, res)
-			} else if res.IsClientAggregates() {
-				aggregateClientAggregates(ctx, res)
+			lsetAttr, _ := res.fields[config.LabelSetAttrName].(string)
+			lset, _ := utils.LabelsFromString(lsetAttr)
+			currentResultHash := lset.Hash()
+
+			// Aggregating cross series aggregates, only supported over raw data.
+			if ctx.isCrossSeriesAggregate {
+				lastTimePerMetric[currentResultHash], lastValuePerMetric[currentResultHash], _ = aggregateClientAggregatesCrossSeries(ctx, res, lastTimePerMetric[currentResultHash], lastValuePerMetric[currentResultHash])
+				// Aggregating over time aggregates
```

Comment:
What's this comment doing here at the end of a block?

Reply:
it's supposed to be for the `else`, but for some reason this is how go formatted it

Reply:
So move it inside the else block?